ext/opentelemetry-ext-dbapi/setup.cfg

```diff
@@ -39,7 +39,7 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api >= 0.4a0
+    opentelemetry-api >= 0.4a1
```

Comment:
Dependencies could technically be `>=0.4a0` or `~=0.4a` since the API shouldn't change between micro versions, but I thought it would be easier to keep all package in sync. I'm interested to hear others' thoughts on this.